### ISSUE TITLE
Image alignment in large gallery

### ISSUE THREAD

**Issue by monahakk** [BCLibraries/burns-exhibits#3]
Is it possible for us to be able to choose the alignment (ie left, right, center) for images in the large gallery pages? Rather than always having the text to the left of the image?

**Comment by Otterfan** [comment]
Totally doable. It will probably look something like:

    {{% largefigure [...] side="left" %}}

**Comment by Otterfan** [comment]
I added a new (optional) `align` attribute, e.g.:

```
{{% largefigure src="../img.jpg" headline="Foo" size="small"  align="left" %}}
```

Allowable values are:

- `right` (default if empty)
- `left`
- `center`

Centered images will automatically be set to 100%, no matter what their `size` attribute.

**Comment by Otterfan** [comment]
Closed in 369ed00, I think.
